brave/brave-core
# Fix toolbar and title not being displayed

## Resolves https://github.com/brave/brave-browser/issues/32222

⚠️ Note: change base branch from `simone/nft-section-bar` to `master` as soon as it gets merged. 

Switch from `Toolbar` to `MaterialToolbar`.
Center `Wallet` title.


![image (9)](https://github.com/brave/brave-core/assets/3308503/dff0c718-6728-43b0-b2b8-5274b49a80b2)




## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Navigate to Brave Wallet
* Observe the title `Wallet` is displayed in the toolbar.
